### DISCUSSION TITLE
What type of CD-R do you use?  Burners, software - etc  // Error Disk Type D code 12

### DISCUSSION THREAD

**Discussion by smvb64** [brad-lin/FreePSXBoot#75] [Q&A] [answered]
Hello all,

Recently installed Tonyhax 1.4.3 freePSXboot on memory card slot2. Able to boot into the exploit perfectly each time.
My console is a PSone  SCPH-101  Bios 4.4.

 The console is able to play retail PS1 games and music CDs with no issue but unable to successfully boot a backup disk. 
Tested loading a retail disk through TonyHax and worked with no issue.

Currently using older  Verbatim  CD-R disks with ImgBurn - dumps are Redump verified. Tried multiple speeds on multiple different burners with no luck.

I may buy a newer batch of Verbatim disks I saw that were recommended. Worst case I may have to adjust the pots which I'm hoping is the absolute last resort

Just curious about which type of disks/burners you guys and used if you have had this issue before?

Best,
Dan

**Comment by rwebster85** [comment, marked as answer]
I've used these discs and Imgburn at the slowest speed for my drive, which is 8x.

Verbatim 43745 52x CD-R DataLifePlus Inkjet Professional - 50pk Spindle https://www.amazon.co.uk/dp/B004S6SIZG/ref=cm_sw_r_apan_glt_fabc_2VWMYXRHF45XB8PK0Z4D?_encoding=UTF8&psc=1

**Reply by smvb64** [reply]
Thank you guys for the replies,
I have been doing some digging and yes not all Verbatim disks are made equal.
Verbatim is just a brand name - the DataLifePlus line are made by Mitsubishi Chemical Corp - the ones I had before were made by  CMC Magnetics Corp. and were truly crap 

 Imgburn will tell you what type of disks you have. 
Just got some DataLifePlus disks, gonna test them out

**Reply by smvb64** [reply]
Works perfectly! Thanks, man. 
Hopefully, this thread can help others in the future - You want DataLifePlus disks  by by Mitsubishi Chemical Corp. They do make a huge difference

**Comment by ghost** [comment]
If you live in the US, you can purchase Verbatim CD-Rs to burn backups and run them without hardly any issues whatsoever on your PS1. You'll need to burn it at 10x speed, which is the lowest it can go, if I recall correctly. They can be purchased at well-known retailers like Walgreens and Wal-Mart, but if you need a bigger supply of CDs, you may want to get in hands with the wholesale pack sold at Sam's Club, if you have a membership.
